#Hacktoberfest: Fixed issue 5376 to improve AWS security group documentation (#5574)
jenkins-infra/jenkins.io: content/doc/tutorials/tutorial-for-installing-jenkins-on-AWS.adoc

. Select your VPC from the list. You can use the default VPC.
. On the **Inbound tab**, add the rules as follows:
.. Select *Add Rule*, and then select *SSH* from the Type list.

> .. Under *Source*, select *Custom*, and in the text box, enter <<Decide who may access your instance,the IP address from step 1 followed by /32 indicating a single IP Address>>. e.g. 104.34.241.123/32 (a single IP address) or 198.51.100.2/24 for a range of 256 IP addresses.

Reviewer: ```suggestion
.. Under *Source*, select *Custom*, and in the text box, enter <<Decide who may access your instance,the IP address from step 1>>, followed by /32 indicating a single IP Address.
For example, 104.34.241.123/32 is a single IP address, while 198.51.100.2/24 results in a range of 256 IP addresses.
```
Thanks so much @ankurkumarz I had a couple suggestions on the update, but definitely think it looks better with the insight you have provided.

I broke it up into 2 sentences, and rearranged some of the wording to flow better.
The only concern I have is that the "24" range is mentioned at the very end of this, but no where else in the page, so it seems to be new information that is not framed as such. It may be worth calling that out specifically if we include the specifics of a single IP address. Or if that is unnecessary information in this process/could lead to other issues, it would be best left out.

Let me know if you have any questions!

Author: Thanks - will continue these points in mind for future considerations.
